### PR TITLE
Chore (Airdrops): Add DEGEN airdrop 2 s2 and s4

### PR DESCRIPTION
**Pull request by OjusWiZard**
## Changes

- [x] Add airdrops `degen2_season2` and `degen2_season4`
- [x] Script to convert CSV to Parquet file
- [x] Some test improvements

### CODE REVIEW

File: scripts/csv_to_parquet.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    scripts/csv_to_parquet.py <path_to_csv>
+"""
+import sys
+import polars
+from pathlib import Path
+
+
+if __name__ == '__main__':
+    data = []
+
```
Comment (yabirgb) [inline scripts/csv_to_parquet.py:14]: you don't need the daata variable here

File: tests/test_airdrops.py [hunk 2]
```diff
@@ -42,8 +42,11 @@ def test_files_and_jsons():
             with open(airdrop['file_path'], 'br') as f:
                 assert airdrop['file_hash'] == hashlib.sha256(f.read()).hexdigest(), f'Invalid hash for {airdrop["file_path"]}'
             df = polars.read_parquet(airdrop['file_path'])
-            for i, (address,) in enumerate(df.select(polars.selectors.by_index(0)).rows()):
-               assert is_checksum_address(address), f'{airdrop["file_path"]} address {address} is not checksummed at row {i+2}'
+            assert df.columns[:2] == ['address', 'amount'], f'{airdrop["file_path"]} does not have address and amount columns'
```
Comment (yabirgb) [inline tests/test_airdrops.py:45]: what happens for files that have a different header?

File: tests/test_airdrops.py [hunk 3]
```diff
@@ -42,8 +42,11 @@ def test_files_and_jsons():
...
+            assert len(df.rows()) > 0, f'{airdrop["file_path"]} is empty'
+            for i, (address, amount) in enumerate(df.select(polars.selectors.by_index(0, 1)).rows()):
+               assert is_checksum_address(address), f'{airdrop["file_path"]} address {address} is not checksummed at row {i+1}'
+               assert float(amount) >= 0, f'{airdrop["file_path"]} amount {amount} is negative at row {i+1}'
```
Comment (yabirgb) [inline tests/test_airdrops.py:49]: 0 amounts shouldn't be in there either right?

Comment (yabirgb) [reply tests/test_airdrops.py:49]: use `from decimal import Decimal`

File: tests/test_airdrops.py (hunk 2 above)
Comment (OjusWiZard) [reply tests/test_airdrops.py:45]: currently they all have the header ['address', 'amount'], and it's also documented that any new airdrop must have those headers
to remind/ensure the structure

**Review by yabirgb — APPROVED** [review]
lgtm

**Pull request merged**